docs/source/changelog.rst

```diff
@@ -7,7 +7,9 @@ Changelog
 - Drop support for Python 3.4 (:issue:`40`)
 - The dataframe passed to ```.to_gbq(...., if_exists='append')``` needs to contain only a subset of the fields in the BigQuery schema. (:issue:`24`)
 - Use the `google-auth <https://google-auth.readthedocs.io/en/latest/>`__ library for authentication because oauth2client is deprecated. (:issue:`39`)
-- ``read_gbq`` now has a ``auth_local_webserver`` boolean argument for controlling whether to use web server or console flow when getting user credentials. Replaces `--noauth_local_webserver` command line argument (:issue:`35`)
+- ``read_gbq`` now has a ``auth_local_webserver`` boolean argument for controlling whether to use web server or console flow when getting user credentials.
+  Replaces `--noauth_local_webserver` command line argument (:issue:`35`)
+- 
```

Comment:
Could we also add an entry to the change log to say that the `verbose` parameter is deprecated and will be removed in a future version?